modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContext.java

```diff
@@ -235,6 +210,48 @@ public String getDefaultEventHandler() {
 				});
 		}
 
+		List<? extends ContentDashboardItemType> contentDashboardItemTypes =
+			_contentDashboardAdminDisplayContext.getContentDashboardItemTypes();
+
+		for (ContentDashboardItemType contentDashboardItemType :
+				contentDashboardItemTypes) {
+
+			labelItemListWrapper.add(
+				labelItem -> {
+					PortletURL portletURL = PortletURLUtil.clone(
+						currentURLObj, liferayPortletResponse);
+
+					Stream<? extends ContentDashboardItemType> stream =
+						contentDashboardItemTypes.stream();
+
+					portletURL.setParameter(
+						"contentDashboardItemTypePayload",
+						stream.filter(
+							curContentDashboardItemType -> !Objects.equals(
+								curContentDashboardItemType.getClassPK(),
+								contentDashboardItemType.getClassPK())
+						).map(
+							curContentDashboardItemType ->
+								curContentDashboardItemType.toJSONString(
+									_locale)
+						).collect(
```

Comment:
Stream may be extended replacing 'toArray' 

`						).map(
							curContentDashboardItemType ->
								curContentDashboardItemType.toJSONString(
									_locale)
						).toArray(String[]::new));
`

Reply:
Very nice!